docs/userguide/modules/getting-started/partials/deploy-db-pgcluster.adoc

```diff
@@ -0,0 +1,41 @@
+== Create a PostgreSQL database instance
+
+The application is going to use to a PostgreSQL database service which
+can be installed using OLM. We will use https://operatorhub.io/operator/postgresql[Crunchy PostgreSQL Operator
+from OperatorHub.io]. We need to make sure that the Operator is
+availabe in our `{quickstart-namespace}` namespace.
+
+The installation of the Operator does not create a database instance. To create a database service instance, you need to create `PostgresCluster` custom resource which will be processed by the Operator.
+
++++ <details><summary> +++
+`PostgresCluster` resource to run database instance
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$pgcluster-deployment.yaml[tag=db-deployment]
+----
++++ </div></details> +++
+
+To create a database instance, create a `PostgresCluster` custom resource by running the following command:
```

Comment:
Suggestion: From here, please apply all the comments given for all the procedures in the Connecting PetClinic application to PostgreSQL database section.

Reply:
thx